tools/travis-setup.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
```

Comment:
From what I can see this file was deleted by your commit:
https://github.com/esl/MongoosePush/commit/93646bb4c0fe12cc2aa601bc05bcfe4e74eb70c9#diff-9facedb7a291effd4d811b201c2d7dae
On master branch there is no such file now - that's why it's all green here

Reply:
We have to remove it then and update the file that replaced this one. I don't remember how it's called. We can't just add this file to repo, since it's probably not even used now (as it was deleted).